social/meeting-transcript/v0.1.0/schema.json

```diff
@@ -0,0 +1,135 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/social/meeting-transcript/v0.1.0/schema.json",
+    "title": "Meeting Transcript",
+    "description": "Schema for a meeting transcript",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#34A853",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "social_meeting_transcript",
+        "indexes": {         
+            "name": ["name"],   
+            "inserted": ["insertedAt"],
+            "date": ["date"],            
+            "source": ["sourceApplication", "sourceId", "calendar_id"]
+        }
+    },
+    "layouts": {
+        "create": ["name", "organizer_email", "participants", "calendar_id"],
+        "view": ["name", "organizer_email", "participants", "date", "duration", "user", "sentence", "summary"]
+    },
+    "allOf": [
+        {
+            "$ref": "https://core.schemas.verida.io/base/v0.1.0/schema.json"
+        },
+        {
+            "type": "object",
+            "properties": {                
+                "organizer_email": {
+                    "title": "Organizer Email",
+                    "description": "Email address of the meeting organizer",
+                    "type": "string",
+                    "format": "email"
+                },
+                "user": {
+                    "title": "User",
+                    "description": "The user who recorded the meeting on behalf of another",
+                    "$ref": "#/$defs/person"
+                },
+                "speakers": {
+                    "title": "Speakers",
+                    "description": "Array of speakers",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/$defs/person"
+                    }
+                },                                
+                "meeting_attendees": {
+                    "title": "Meeting Attendees",
+                    "description": "List of meeting attendees",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/$defs/person"
+                    }
+                },                
+                "duration": {
+                    "title": "Duration",
+                    "description": "Duration of the meeting audio in seconds",
+                    "type": "number"
+                },                
+                "date": {
+                    "title": "Date",
+                    "description": "Epoch time in milliseconds when the transcript was created",
```

Comment:
Make this a `date-time` field and convert it before saving. We don't use epoch times in schemas.

Reply:
ok